src/fides/api/ops/util/saas_util.py

```diff
@@ -111,7 +116,7 @@ def get_collection_after(
     return collection.after
 
 
-def merge_datasets(dataset: Dataset, config_dataset: Dataset) -> Dataset:
+def merge_datasets(dataset: GraphDataset, config_dataset: GraphDataset) -> GraphDataset:
     """
     Merges all Collections and Fields from the config_dataset into the dataset.
```

Comment:
I think this may need to be updated

```suggestion
    Merges all Collections and Fields from the config_dataset into the GraphDataset.
```

Reply:
This code comment to me looks like it's referring to the variable names not the data types of those variable names. I can wrap them in quotes in the docstring -